packages/client/webapp/src/views/TA2Playground.vue

```diff
@@ -323,6 +324,7 @@ export default defineComponent({
 				method: 'GET'
 			});
 			const output = await resp.json();
+			console.log(petrinetValidator(g) ? 'Valid Petrinet' : 'Invalid Petrinet');
```

Comment:
Maybe make it part of the JSON output by adding a boolean attribute `isValidPetriNet` or something equivalient?